client/front-end/package.json

```diff
@@ -13,10 +13,12 @@
     "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\""
   },
   "dependencies": {
+    "apollo-boost": "^0.3.1",
```

Comment:
Packages look good, _but_ looks like we're missing a package.
From the apollo-boost instructions 👇 
First, install apollo-boost. If you don't have graphql & react-apollo already in your project, please install those too.
```npm i apollo-boost graphql react-apollo -S```

Reply:
resolved. Dependency added.